ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -162,14 +162,14 @@ class Metrics(val registry: MetricRegistry) {
           val fetchInputs: Timer = registry.timer(Prefix :+ "fetch_inputs")
           val validate: Timer = registry.timer(Prefix :+ "validate")
           val commit: Timer = registry.timer(Prefix :+ "commit")
-          val transformSubmission: Timer = registry.timer(prefix :+ "transform_submission")
+          val transformSubmission: Timer = registry.timer(Prefix :+ "transform_submission")
 
-          val acquireTransactionLock: Timer = registry.timer(prefix :+ "acquire_transaction_lock")
+          val acquireTransactionLock: Timer = registry.timer(Prefix :+ "acquire_transaction_lock")
           val failedToAcquireTransaction: Timer =
-            registry.timer(prefix :+ "failed_to_acquire_transaction")
-          val releaseTransactionLock: Timer = registry.timer(prefix :+ "release_transaction_lock")
+            registry.timer(Prefix :+ "failed_to_acquire_transaction")
+          val releaseTransactionLock: Timer = registry.timer(Prefix :+ "release_transaction_lock")
 
-          val stateValueCache = new CacheMetrics(registry, prefix :+ "state_value_cache")
+          val stateValueCache = new CacheMetrics(registry, Prefix :+ "state_value_cache")
```

Comment:
We use lowercase `prefix` everywhere else in this file, as far as I can see. Can you please make it consistent?

Otherwise you're just going to cause bugs like this one. For example, `openEnvelope`, directly above, which you missed.

Reply:
Good catch. According to the [Scala style guide](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods) constants should be in upper camel case. Shouldn't we rather switch everywhere to `Prefix`?
This code has been moved from a repo where we follow the Scala style guide hence the inconsistencies.

Reply:
No opinion. I suggest making it internally consistent in this PR and then, if you're so inclined, opening another one when it's merged to make it externally consistent.

Reply:
OK, will make a separate PR for addressing the naming conventionissue.